tests/unit/test_reader_v1_arrow.py

```diff
@@ -178,9 +183,7 @@ def test_to_arrow_w_scalars_arrow(class_under_test, mock_gapic_client):
     assert actual_table == expected_table
 
 
-@pytest.mark.skipif(
-    PANDAS_INSTALLED_VERSION >= pkg_resources.parse_version("2.0.0"), reason=""
-)
+@pytest.mark.skipif(PANDAS_INSTALLED_VERSION >= "2.0.0", reason="")
```

Comment:
```
>>> version1 = "10.0.0"
>>> version2 = "2.0.0"
>>> version1>version2
False
```
The above check won't work once we get into double digits. Something like `PANDAS_INSTALLED_VERSION[0:2] not in ["0.", "1."]` could work. We could also convert the first part of the version string to an integer an then do a comparison on that. Something like `int(re.findall("\d+", PANDAS_INSTALLED_VERSION)[0])` or `int(version1.split(".")[0])`. 

I think `PANDAS_INSTALLED_VERSION[0:2] not in ["0.", "1."]` would be the easiest to read